x-pack/plugins/observability_solution/apm/server/routes/assets/services/get_services_from_assets.ts

```diff
@@ -0,0 +1,97 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { Logger } from '@kbn/core/server';
+import { errors } from '@elastic/elasticsearch';
+import { WrappedElasticsearchClientError } from '@kbn/observability-plugin/server';
+import { termQuery, rangeQuery, kqlQuery } from '@kbn/observability-plugin/server';
+import { ASSET_TYPE, FIRST_SEEN, LAST_SEEN } from '../../../../common/es_fields/assets';
+import { AssetsESClient } from '../../../lib/helpers/create_es_client/create_assets_es_client/create_assets_es_clients';
+import { withApmSpan } from '../../../utils/with_apm_span';
+import { ServiceAssetDocument } from './types';
+
+export const MAX_NUMBER_OF_SERVICES = 1_000;
+
+type AssetType = 'host' | 'service';
```

Comment:
Should we include `host` in this iteration?

Reply:
https://github.com/elastic/kibana/pull/181842/commits/4f4432fba919e9b3b18a494a957676c47695fde5#diff-6817284b7b024c8c9666e7ca740e44bf8c06cd7c36602ac0ce379ab26a1ad6a0R19 cleaned it up